real_preprocessing/README.txt

```diff
@@ -0,0 +1,48 @@
+Additional packages required for real_preprocessing: 
+
+	apriltag_ros (http://wiki.ros.org/apriltag_ros)
+	cv_bridge (http://wiki.ros.org/cv_bridge)
+	tf_conversions (http://wiki.ros.org/tf_conversions)
+
+Must be included  in package.xml as dependencies and CMakelists.txt.
```

Comment:
Not required, since your package CMakeLists already includes this and if someone pulls this repo it will already be set up.